cs/src/comm/epoxy-transport/EpoxyNetworkStream.cs

```diff
@@ -369,19 +365,12 @@ public void Shutdown()
             }
         }
 
-        static void ShutdownSocketSafe(Socket socket, EpoxyNetworkStream epoxyStream, Logger logger)
+        static void SafeShutdownSocket(Socket socket, Logger logger)
         {
-            if (epoxyStream != null)
-            {
-                epoxyStream.Shutdown();
```

Comment:
After the refactoring, there's no longer a failure mode that has an unowned, non-null `EpoxyNetworkStream`.

The next thing we do--both client- and server-size--after calling `MakeAsync` is start the Epoxy handshake by creating an `EpoxyConnection` (this connection object now owns the `EpoxyNetworkStream`) and calling `EpoxyConnection.StartAsync()`. If the Epoxy handshake fails, the connection object will shut down the `EpoxyNetworkStream` it owns.